test/Hyperledger.Aries.Tests/tester

```diff
@@ -0,0 +1,2 @@
+#! /bin/zsh
```

Comment:
Move this script to `scripts` folder instead? Maybe at `.sh` extension to it?

Reply:
Done